docker-compose.yml

```diff
@@ -206,6 +206,16 @@ services:
       OPG_LPA_COMMON_REDIS_CACHE_URL: tcp://redis:6379
 
       OPG_LPA_OS_PLACES_HUB_ENDPOINT : "https://api.os.uk/search/places/v1/postcode"
+    healthcheck:
```

Comment:
Might be good to just use a script which wraps all this up to reduce verbosity, if possible.